docs/roles/cp4d_db2wh_restore.md

```diff
@@ -0,0 +1,65 @@
+cp4d_db2wh_restore
+==========
+
+This role runs a restore procedure onto a CloudPak for Data DB2 Warehouse instance.
+In order to begin the restore process, you must have the required files to run a successful restore process in the chosen `DB2WH_BACKUP_DIR`:
+
+- DB2 backup files i.e `BLUDB.0.db2inst1.DBPART000.202XXXXXXXXXXX.001`
+- DB2 keystore files (.p12 and .sth)
+- DB2 instance master key label file (.kdb)
+
+Note: These files are generated automatically if you run `ibm.mas_devops.cp4d_db2wh_backup` role. If any of the above files are not found in the `DB2WH_BACKUP_DIR`, the restore process will fail.
+
+Role Variables
```

Comment:
Can you update the README to match the new(ish) format we are using now: e.g. https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_db2wh/README.md#role-variables

Reply:
Fixed.